app/models/incoming_message.rb

```diff
@@ -79,6 +79,15 @@ class IncomingMessage < ApplicationRecord
   delegate :message_id, to: :raw_email
   delegate :multipart?, to: :raw_email
   delegate :parts, to: :raw_email
+  delegate :legislation, to: :info_request
+
+  def legislation_references
+    legislation.find_references(get_main_body_text_folded)
+  end
+
+  def refusals
+    legislation_references.select(&:refusal?)
```

Comment:
Nether of these have specs.

I think we can make `legislation_references` private for now at least since we're only using it internally.

Really, we just want a simple integration spec along the lines of:

Given a message with content X, we we should identify N number of refusals.